src/routes/quantity/update.ts

```diff
@@ -8,6 +8,7 @@ interface Request {
   };
   body: {
     name?: string;
+    color?: string;
```

Comment:
Same 'o

Reply:
but this is `update`, each field is optional here, or may be I do not understand you completely ?

In `add` I've removed `?`

Reply:
Sorry, you are absolutely right!